### ISSUE TITLE
SimpleUpdate and Target uris ?

### ISSUE THREAD

**Issue by frsauvage** [dell/redfish-ansible-module#84]
Hi,
Is there a standard way to retrieve uris from Systems (part) that should be updated in SimpleUpdate/Targets ?
I mean from one side I have an inventory like :
- Name = Component Name 
- Version = Component Version
- @odata.id = xxx-Name-xxx-Version-xxx-Installed-xxx or idem-Available-idem
 
From the other side, I have component instances inside Systems part

 => So question is : should the "Name" of the Instance Name from Systems part to be equalized with inventory Name of the SoftwareInventory part ?
In other words, the target uris are /Systems/xxx uris, so how to retrieve the appropriate component instance uris that should really be updated with the ImageUri  Name/Version/Available image ?

I tried to summarize, not sure to be so clear, do not hesitate pour ask me details if necessary,

Best Regards,
Francine

**Comment by jose-delarosa** [comment]
@frsauvage SimpleUpdate? Is this part of the Redfish spec?

**Comment by frsauvage** [comment]
Thank a lot, @jose-delarosa, Yes it is, SimpleUpdate is the only action for UpdateService : 
see  https://redfish.dmtf.org/schemas/UpdateService.v1_4_0.json
